app/models/admin/collection.rb

```diff
@@ -77,16 +75,12 @@ class Admin::Collection < ActiveFedora::Base
 
   has_subresource 'poster', class_name: 'IndexedFile'
 
-  around_save :reindex_members, if: Proc.new{ |c| c.name_changed? or c.unit_changed? }
-  around_save :return_checkouts, if: Proc.new{ |c| c.cdl_enabled_changed? && c.cdl_enabled == false }
+  around_save :reindex_members, if: Proc.new { |c| c.name_changed? or c.unit_changed? }
```

Comment:
We'll need to think about how this trigger needs to change since the collection no longer knows when the unit's name changes.  We would change this to be just `c.name_changed?` and have a similar `around_save` in Unit which triggers `reindex_members` on all of its collections.  This could be a _lot_ of reindexing so we may want a warning.

This doesn't need to be handled in this PR.